tests/harness_test.go

```diff
@@ -197,11 +206,11 @@ func getAttestations(t *testing.T) (string, map[int]string) {
 	return file, attestations
 }
 
-func saveAttestation(t *testing.T, attestation string) {
-	file, attestations := getAttestations(t)
+func saveEntry(t *testing.T, entry StoredEntry) {
+	file, attestations := getEntries(t)
 	logIndex := activeTreeSize(t) - 1
```

Comment:
is this a safe assumption? we could update `getUUIDFromUploadOutput` to extract the actual index and set it in `StoredEntry` and not have to assume here.

Reply:
sure, done!